Dockerfile

```
@@ -8,9 +8,7 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
 	; echo 'Binary::apt::APT::Keep-Downloaded-Packages "true";' > /etc/apt/apt.conf.d/keep-cache \
 	&& apt-get update \
 	&& apt-get install -yqq --no-install-recommends \
-	build-essential wget ca-certificates \
-	&& wget https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -O /usr/bin/yq \
-	&& chmod +x /usr/bin/yq
+	build-essential wget ca-certificates
```

```suggestion
	build-essential
```

`wget` と `ca-certificates` も `yq` コマンドインストールのために必要なパッケージなので、削除して問題なさそうです。